### DISCUSSION TITLE
Error: No such keg: /usr/local/Cellar/php

### DISCUSSION THREAD

**Discussion by chansnock** [shivammathur/homebrew-php#2553] [Q&A / Help]
Hi, i was install php and Error: No such keg: /usr/local/Cellar/php

I want to update my php on macos catalina to php 7.4

**Comment by shivammathur** [comment]
@chansnock 

macOS Catalina is not supported anymore by Apple, Homebrew and this project, please consider upgrading your OS to Monterey (12) or newer if possible.

**Reply by chansnock** [reply]
ok thank's @shivammathur  for information
